### PR TITLE
[DOCFIX]Add java doc for PermissionCheckTest

### PR DESCRIPTION
**Pull request by Jason0812**
Add java doc comments for PermissionCheckTest:
* verifyRead
* verifyGetFileId

### CODE REVIEW

**Comment by Jason0812** [comment]
alluxio-bot,check this please

**Comment by Jason0812** [comment]
alluxio-bot, check this please

**Comment by Jason0812** [comment]
alluxio-bot, check this please

File: core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java [hunk 1]
```diff
@@ -615,13 +615,26 @@ private String createUnreadableFileOrDir(boolean isFile) throws Exception {
     }
   }
 
+  /**
+   * This method verifies the read permission
```
Comment (yupeng9) [inline core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java]: Per checkstyle, you need a dot at the end.

File: core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java [hunk 2]
```diff
@@ -615,13 +615,26 @@ private String createUnreadableFileOrDir(boolean isFile) throws Exception {
...
+   */
   private void verifyRead(TestUser user, String path, boolean isFile) throws Exception {
     try (Closeable r = new AuthenticatedUserRule(user.getUser()).toResource()) {
       verifyGetFileId(user, path);
       verifyGetFileInfoOrList(user, path, isFile);
     }
   }
 
+  /**
+   * This method verifies the get fileId
```
Comment (yupeng9) [inline core/server/master/src/test/java/alluxio/master/file/PermissionCheckTest.java]: Per checkstyle, you need a dot at the end.

**Comment by Jason0812** [comment]
alluxio-bot,check this please

**Comment by Jason0812** [comment]
alluxio-bot, check this please

**Comment by Jason0812** [comment]
@yupeng9 change finished as you advised, please help to review

**Pull request merged**